plotly/offline/offline.py

```diff
@@ -129,6 +129,8 @@ def init_notebook_mode(connected=False):
         script_inject = (
             ''
             '<script>'
+            'window.is_loaded = false;'
+            'setTimeout(function(){{window.is_loaded=true}},100);'
```

Comment:
^^ That sounds better than adding a property onto `window`. Want to give that a shot?